micropub.php

```diff
@@ -810,31 +810,44 @@ private static function check_error( $result ) {
 			return $result;
 		}
 
+
+		public static function get_micropub_endpoint() {
+			return apply_filters( 'micropub_endpoint', site_url( '?micropub=endpoint' ) );
+		}
+
+		public static function get_authentication_endpoint() {
+			return apply_filters( 'micropub_authentication_endpoint', MICROPUB_AUTHENTICATION_ENDPOINT );
+		}
+
+		public static function get_token_endpoint() {
+			return apply_filters( 'micropub_endpoint', MICROPUB_TOKEN_ENDPOINT );
```

Comment:
...which is a good sign that it's probably time for unit tests for these headers. :P consider adding some in this PR?